Assignment_1/palindrome_permutation.js

```diff
@@ -0,0 +1,33 @@
+const palindromePermutation = (str) => {
+
+    const charMap = new Map(); //frequency of each char in string
+    str = str.replace(/\s/g, '').toLowerCase();
+    //remove spaces and convert string to lower case
+  //
+    for (const char of str) {
+        if (charMap.has(char)) {
+          charMap.set(char, charMap.get(char) + 1);
+        } else {
+          charMap.set(char, 1);
+        }
+      }
+
+    //checking palindrome
+    let count = 0;
+    for (const i of charMap.values()) {
+        if (i % 2 !== 0) {
+        count++;
+        }
+    }
+    //if one characater with odd freq
+    return count <=1;
+
+}
```

Comment:
Again, watch your spacing here. The convention in JS is to not have extra lines of whitespace before a closing bracket.